package.json

```
@@ -69,18 +70,19 @@
     "eslint-plugin-jest": "^27.6.0",
     "husky": "^8.0.3",
     "jest": "^29.7.0",
+    "jest-light-runner": "^0.6.0-alpha.1",
     "lint-staged": "^15.0.2",
     "np": "^8.0.4",
     "npm-run-all": "^4.1.5",
     "prettier": "^3.0.3",
     "remark-cli": "^12.0.0",
-    "stylelint": "^15.11.0",
+    "stylelint": "^16.0.0-2",
```

```suggestion
    "stylelint": "^16.0.0-2",
```

We'll need to bump this to non-prerelease before releasing.

---

Fixed via e68db75